data_validation/__main__.py

```diff
@@ -225,6 +225,8 @@ def build_config_from_args(args: Namespace, config_manager: ConfigManager):
         config_manager (ConfigManager): Validation config manager instance.
     """
 
+    # If configuring generate-table-partitions, validation_type will be set to underlying validation either Row
+    # or custom query and, if custom query - args.custom_query_type will also be set.
```

Comment:
This statement would be better placed as part of the function's documentation, since it's not a direct comment for a code line or block

Reply:
Fixed